packages/backend/src/peripherals/database/types.ts

```diff
@@ -261,12 +268,14 @@ declare module 'knex/types/tables' {
     finality: FinalityRow
     tracked_txs_configs: TrackedTxsConfigRow
     l2_costs: L2CostsRow
+    l2_costs_prices: L2CostsPricesRow
```

Comment:
why this change?

Reply:
I missed this before when the the table was being added